en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md

```diff
@@ -0,0 +1,66 @@
+# Update Choreo Connect
+
+Choreo Connect components are available as public docker images in [WSO2](https://hub.docker.com/u/wso2) organization in the docker hub whereas you will be able to find images corresponding to [Choreo Connect releases](https://github.com/wso2/product-microgateway/releases).
+
+However, if you have a WSO2 Subscription you will be able to use latest docker images with updates, which includes bug fixes, features, improvements, etc. on top of genarally available releases. This documentation is about how you can use that particular updates.
+
+## Docker Image Tags
+
+The docker images related to Choreo Connect updates are reside in [WSO2 private docker registry](https://docker.wso2.com/) and the image tags are having the following format for Choreo Connect components.
+
+<table>
+    <thead>
+        <tr>
+            <th>Component Name</th>
+            <th>Image tag format</th>
+            <th>Example</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <a href="{{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview/#router">Router</a>
+            </td>
+            <td>
+                <code>choreo-connect-router:{WSO2_PRODUCT_VERSION}.{UPDATE_LEVEL}</code>
+            </td>
+            <td>
+                <code>choreo-connect-router:1.1.0.3</code>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <a href="{{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview/#enforcer">Enforcer</a>
+            </td>
+            <td>
+                <code>choreo-connect-enforcer:{WSO2_PRODUCT_VERSION}.{UPDATE_LEVEL}</code>
+            </td>
+            <td>
+                <code>choreo-connect-enforcer:1.1.0.3</code>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <a href="{{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview/#adapter">Adapter</a>
+            </td>
+            <td>
+                <code>choreo-connect-enforcer:{WSO2_PRODUCT_VERSION}.{UPDATE_LEVEL}</code>
+            </td>
+            <td>
+                <code>choreo-connect-adapter:1.1.0.3</code>
```

Comment:
Shall we give the complete image name?
```suggestion
                <code>docker.wso2.com/choreo-connect-adapter:1.1.0.3</code>
```

Reply:
If it makes the table row lengthy we can give one example so the user know how it would look like after updating.
```
enforcer:
    image: docker.wso2.com/choreo-connect-enforcer:1.1.0.3
```

Reply:
Provided an example, as it makes table lengthy, when we add the above..